src/chrome/content/rules/Sunrise.xml

```diff
@@ -1,14 +1,67 @@
-<ruleset name="Sunrise">
-    <target host="sunrise.ch" />
-    <target host="*.sunrise.ch" />
+<!--
+	Nonfunctional hosts in *.sunrise.ch:
+		- b2b.sunrise.ch (m)
+		- campaign.sunrise.ch (i)
+		- corporate.sunrise.ch (m)
+		- cpbx.sunrise.ch (m)
+		- emarketing.sunrise.ch (i)
+		- itunes.sunrise.ch (m)
+		- kobik.sunrise.ch (t)
+		- m.sunrise.ch (m)
+			- business.marketing.sunrise.ch (i)
+		- mobile.sunrise.ch (s)
+			- www.mobile.sunrise.ch (s)
+		- partners.sunrise.ch (s)
+		- survey.sunrise.ch (i)
+		- speed.sunrise.ch (s)
+		- testimonials.sunrise.ch (m)
+
+	Fetch issues:
+		- codes.sunrise.ch
+		- www.partnerdeal.sunrise.ch
 
-    <!-- www. redirects to valid www1. -->
-    <exclusion pattern="^http://www\.sunrise\.ch" />
+	h: http redirect
+	i: incomplete cert chain
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Sunrise">
+	<target host="sunrise.ch" />
+	<target host="www.sunrise.ch" />
+	<target host="autodiscover.sunrise.ch" />
+	<target host="cct.sunrise.ch" />
+	<target host="cockpit.sunrise.ch" />
+		<target host="www.community.sunrise.ch" />
+		<target host="home.cpbx.sunrise.ch" />
+		<target host="userportal.cpbx.sunrise.ch" />
+	<target host="e-deliverymobile.sunrise.ch" />
+	<target host="ebill.sunrise.ch" />
+	<target host="esign.sunrise.ch" />
+	<target host="event1.sunrise.ch" />
+	<target host="event2.sunrise.ch" />
+	<target host="internet.sunrise.ch" />
+		<test url="http://internet.sunrise.ch/iff/index_fr.html" />
+	<target host="jobs.sunrise.ch" />
+	<target host="lss.sunrise.ch" />
+	<target host="maps.sunrise.ch" />
+	<target host="mip.sunrise.ch" />
+	<target host="mobilepbx.sunrise.ch" />
+	<target host="msm.sunrise.ch" />
+	<target host="olympus.sunrise.ch" />
+		<target host="www.recommend.sunrise.ch" />
+	<target host="salesportal.sunrise.ch" />
+	<target host="sbp.sunrise.ch" />
+	<target host="suota.sunrise.ch" />
+	<target host="travel.sunrise.ch" />
+		<target host="serviceportal.vpbx.sunrise.ch" />
+	<target host="webmail.sunrise.ch" />
+	<target host="webmail1.sunrise.ch" />
+	<target host="webmail2.sunrise.ch" />
+	<target host="www1.sunrise.ch" />
 
-    <securecookie host="^(?:.*\.)?sunrise\.ch$" name=".+" />
+	<securecookie host="^(?:.*\.)?sunrise\.ch$" name=".+" />
```

Comment:
AFIAK, this can be replaced with `.+` without affecting the ruleset behavior

Reply:
Updated